### ISSUE TITLE
Cannot run macos ventura on plasma 6

### ISSUE THREAD

**Issue by CantFindDev** [quickemu-project/quickemu#1287]
# Expected behavior

Run the vm

## Actual behavior

Does not run the vm for some reason

## Quickemu output

<details>
  <summary>Quickemu output</summary>

  ```text
cat: macos-ventura/macos-ventura.pid: No such file or directory
 - Process:  Starting macos-ventura.conf as macos-ventura ()

  ```
</details>

## System Information ##

OS: Arch Linux x86_64 
Host: ABRA A5 V9.3 
Kernel: 6.9.5-arch1-1 
Resolution: 1920x1080 
DE: Plasma 6.0.5 
WM: KWin 
CPU: Intel i5-7300HQ (4) @ 3.500GHz 
GPU: NVIDIA GeForce GTX 1050 Ti Mobile 
GPU: Intel HD Graphics 630

**Comment by lj3954** [comment]
Please install the latest quickemu from git. v4.9.4 has a known issue with invalid CPU flags being passed to QEMU for macOS VMs.

**Comment by PixsaOJ** [comment]
Is `quickgui-bin` 1.2.8-1 supposed to work from aur?

**Comment by lj3954** [comment]
Hello. That shouldn't be a problem, just replace the quickemu AUR package with the quickemu-git AUR package until v4.9.5 is released.

**Comment by PixsaOJ** [comment]
That seems to start the machine, but SPICE connection is not establishing. There is also no listener on `5930` PORT.

SSH is listening on `22220` but i cant log in.

It shuts down after couple minutes, so this makes sense. Could this be my issue? I didn't really touch any configuration.

Here is git version btw, still `4.9.4...`

![image](https://github.com/quickemu-project/quickemu/assets/12684017/2648ee3f-3c0f-4b3e-b68d-2edc42831f06)

**Comment by PixsaOJ** [comment]
ran it without spice ✅

**Comment by lj3954** [comment]
That's also a known issue, and the fix isn't yet merged. Check #1264

**Comment by flexiondotorg** [comment]
FIxed via #1264
